I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 80, in script
    jump two_one
ScriptError: could not find label 'two_one'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 80, in script
    jump two_one
  File "/Applications/renpy-7.1.3-sdk/renpy/ast.py", line 1633, in execute
    rv = renpy.game.script.lookup(target)
  File "/Applications/renpy-7.1.3-sdk/renpy/script.py", line 875, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'two_one'.

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 7.1.3.1092
Blind Run 1.0
Sat Feb  2 13:09:00 2019
